armada/armada.py

```diff
@@ -24,16 +24,14 @@ def __init__(self, args):
 
         self.tiller = Tiller()
 
-
     def find_release_chart(self, known_releases, name):
         '''
         Find a release given a list of known_releases and a release name
         '''
-        for chart_name, version, chart, values in known_releases:
+        for chart_name, _, chart, values in known_releases:
```

Comment:
If it's not used, it should be removed entirely instead of just hanging as a "dont care" variable.  Also, do we want to consider situations where we would care about to version?  For example, if a chart version is deprecated.  I can see situations where we would want to return specific information on deprecated versions of a release.